diagnostics/test_canary.md

```diff
@@ -0,0 +1,13 @@
+# Test Feature in Canary
+
+The latest runtime changes are first shipped in the WebView2 runtime in the Edge Canary Browser. If you would like to test the new changes or help us to verify that your issue is fixed with your application, here's a suggested way to do so:
+
+1. Download [Edge Canary](https://www.microsoft.com/en-us/edge/download/insider)
+2. Run the following command in a PowerShell with Administrative Priviledges:  
+  `REG ADD HKCU\Software\Policies\Microsoft\Edge\WebView2\ChannelSearchKind /v <your_app.exe> /t REG_DWORD /d 1`
+3. Restart your application, and you should be able to see that your application picks up the latest runtime. 
+
+To remove the registry key added, run the following:  
+`REG DELETE HKCU\Software\Policies\Microsoft\Edge\WebView2\ChannelSearchKind`
```

Comment:
the command line for adding regkey is for particular app, but removing one is for all?

Reply:
hmm I was trying to make it simple. Let me edit it

Reply:
can do both - or mention that it will remove others. I agree about making it simple

Reply:
I removed it